### PR TITLE
CI should pull logs if test fails

### PR DESCRIPTION
**Pull request by cblecker**
Pull logs if a test fails (like Jenkins)
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-charts-gce/17407

### CODE REVIEW

File: test/helm-test/main.go [hunk 1]
```diff
@@ -252,7 +252,11 @@ func doMain() int {
 		xmlWrap(fmt.Sprintf("Helm Test %s", path.Base(chartPath)), func() error {
 			o, execErr := output(exec.Command(helmPath, "test", rel))
 			if execErr != nil {
-				return fmt.Errorf("%s Command output: %s", execErr, string(o[:]))
+				l, logsErr := output(exec.Command(kubectlPath, "logs", rel, "--namespace", ns))
```
Comment (viglesiasce) [inline test/helm-test/main.go:255]: This does `kubectl logs $RELEASE` but that wont necessarily map to any pods afaik.

Comment (viglesiasce) [reply test/helm-test/main.go:255]: @mattfarina did a similar thing for the pre-submit tests here:
https://github.com/kubernetes/charts/blob/master/test/changed.sh#L48

**Review by viglesiasce — CHANGES_REQUESTED** [review]

File: test/helm-test/main.go (hunk 1 above)
Comment (mattfarina) [reply test/helm-test/main.go:255]: This isn't going to work if a pod has more than one container. logs needs to be fetched for each container in a pod.

**Review by mattfarina — CHANGES_REQUESTED** [review]
Getting the logs would have helped a few days ago when we were digging into an error. I like the idea. There are just a couple problems with the implementation.

**Comment by unguiculus** [comment]
Fixed with the new chart testing lib.

**Comment by cblecker** [comment]
Don't have the cycles to address this, so I'm closing. Probably still a good idea to pull pod logs, however.

**Pull request closed without merging**